test cases/frameworks/6 gettext/data3/meson.build

```diff
@@ -0,0 +1,36 @@
+
+if itstool.found()
+
+
+mi_translated = i18n.itstool_join(
+  input:  'com.mesonbuild.test.intlprog.metainfo.xml',
+  output: 'com.mesonbuild.test.intlprog.metainfo.xml',
+  mo_targets: mo_targets,
+  its_files: ['metainfo.its'],
+  install: true,
+  install_dir: join_paths(get_option('datadir'), 'metainfo'),
```

Comment:
This is old-style code, the modern equivalent should be `get_option('datadir') / 'metainfo'`.